lib/rubocop/cop/layout/heredoc_argument_closing_parenthesis.rb

```diff
@@ -215,6 +216,19 @@ def incorrect_parenthesis_removal_end(node)
           end
         end
 
+        def exist_argument_between_heredoc_end_and_closing_parentheses?(node)
```

Comment:
Why is the end of the heredoc significant here?

Reply:
Without this check, some existing tests will fail. By the way, "the end of the heredoc" points to `EOS`. (It may not be good choice for the name...)

```ruby
<<~EOS
  text
EOS <- the end of the heredoc
```